fluentd-forward/README.md

```diff
@@ -0,0 +1,341 @@
+# Fluentd-forward
+
+This plugin is used to export Audit Log events to Fluentd service.
+
+## Setup
+
+### Prerequisites
+
+This guide assumes that you have:
+* Teleport 6.2 or newer
+* Admin privileges to access tctl
+* OpenSSL or LibreSSL (macOS)
+* Docker to build plugin from source and run fluentd example instance
+
+The required Fluentd version for production setup is v1.12.4 or newer. Lower versions do not support TLS.
+
+### <a name="user"></a>Create user and role for access audit log events
+
+Log into Teleport Authentication Server, this is where you normally run tctl. Create a new user and role that only has read-only API access to the `event` API. The below script will create a [yaml resource file](example/fluentd-forward.yaml) for a new user and role.
+
+```yaml
+kind: user
+metadata:
+  name: fluentd-forward
+spec:
+  roles: ['fluentd-forward']
+version: v2
+---
+kind: role
+metadata:
+  name: fluentd-forward
+spec:
+  allow:
+    rules:
+      - resources: ['event']
+        verbs: ['list','read']
+version: v3
+```
+
+Here and below follow along and create yaml resources using `tctl create -f`:
+
+```sh
+tctl create -f fluentd-forward.yaml
+```
+
+### Export fluentd-forward Certificate
+
+Teleport Plugin use the fluentd role and user to read the events. We export the identity files, using tctl auth sign.
+
+```sh
+tctl auth sign --format=tls --user=fluentd-forward --out=fd --ttl=720h
```

Comment:
will this work with format=file?

Reply:
Yes, it is shown and described below in Cloud section.